README

```diff
@@ -108,3 +108,9 @@ However if parameter is very insensitive ELO-wise, one needs to use a larger val
 Also if one has even some sort of idea about the ELO-sensitivity of the parameter and how far from optimum
 it might be at maximum, one can use the built-in simulator, to find good values for Rk and ck.
 See files 'simul_1var.conf' and 'simul_1var.var' as an example.
+
+By using the built-in simulator, we've had success with the following values:
```

Comment:
Everything is explained in the forum:
https://groups.google.com/forum/?fromgroups=#!topic/fishcooking/iyAFOdVx4RE%5B101-125%5D